### PR TITLE
Portfolio - AWN case

### PR DESCRIPTION
**Pull request by galdranorn**

### CODE REVIEW

File: _sass/new-styles/modules/portfolio/awn/_goal.scss [hunk 1]
```diff
@@ -0,0 +1,88 @@
+.portfolio-goal--awn {
+
```
Comment (siaradona) [inline _sass/new-styles/modules/portfolio/awn/_goal.scss:2]: remove blank line

File: _sass/new-styles/modules/portfolio/awn/_goal.scss [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    .portfolio__quote .icon-quote {
+      top: -30px;
+      left: -35px;
+    }
+
+    .portfolio__columns--right img {
+      border-radius: 8px;
+    }
+  }
+}
```
Comment (siaradona) [inline _sass/new-styles/modules/portfolio/awn/_goal.scss:34]: add blank line

File: _sass/new-styles/modules/portfolio/awn/_goal.scss [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+@media (min-width: 1200px) {
+  .portfolio-goal--awn {
+    .portfolio__quote {
+
+
+      .icon-quote {
+
```
Comment (siaradona) [inline _sass/new-styles/modules/portfolio/awn/_goal.scss:84]: ?

Comment (galdranorn) [reply _sass/new-styles/modules/portfolio/awn/_goal.scss:84]: 👍 shame

**Pull request merged**